go.mod

```diff
@@ -274,6 +274,3 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator
 
 // Replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheus with a fork until https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/24288 is merged.
 replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheus => github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheus v0.0.0-20230718193318-2bc9904b7ae8
-
-// Use a fork of weaveworks/common while we work out if there is a better design for https://github.com/weaveworks/common/pull/293
-replace github.com/weaveworks/common => github.com/weaveworks/common v0.0.0-20230714173453-d1f8877b91ce
```

Comment:
Note to reviewers: removed because https://github.com/weaveworks/common/pull/299 has been merged.